packages/task/src/browser/task-configurations.ts

```diff
@@ -179,8 +200,58 @@ export class TaskConfigurations implements Disposable {
         this.tasksMap.delete(source);
     }
 
-    getTaskCustomizations(type: string): TaskCustomization[] {
-        return this.taskCustomizations.filter(c => c.type === type);
+    removeTaskCustomizations(configFileUri: string) {
```

Comment:
In general I guess we should try and document the `public` methods.

Reply:
Not part of your change, but do you believe `task-configuration.ts` should also be unit tested?
I believe it is possible since there are no direct UI dependencies.

Reply:
yes adding tests to `task-configuration.ts` would be helpful. I am going to make more changes to this file (hopefully in the next pr) and will add tests after.